test/library/tomography/test_state_tomography.py

```diff
@@ -191,6 +195,23 @@ def test_exp_circuits_measurement_indices(self, meas_qubits):
         fid = qi.state_fidelity(target_state, qi.Statevector(target_circ))
         self.assertGreater(fid, 0.99, msg="target_state is incorrect")
 
+    def test_circuit_serialization(self):
+        """Test simple circuit serialization"""
+        circ = QuantumCircuit(2)
+        circ.h(0)
+        circ.s(0)
+        circ.cx(0, 1)
+
+        exp = StateTomography(circ)
+        circs = exp.circuits()
+
+        qpy_file = io.BytesIO()
+        qpy.dump(circs, qpy_file)
+        qpy_file.seek(0)
+        new_circs = qpy.load(qpy_file)
```

Comment:
Beyond scope of this PR, but we should probably add similar tests for serialization of generated circuits for all experiments.

Reply:
I opened https://github.com/Qiskit/qiskit-experiments/issues/1063 to keep track of this.

Reply:
I think you can probably use the round trip json serialization test function (like we use for experiment results), since json serializer should serialize a list of circuits via qpy.